dropshot_endpoint/src/lib.rs

```diff
@@ -53,6 +54,15 @@ struct Metadata {
 
 const DROPSHOT: &str = "dropshot";
 
+const ERR_MSG_USAGE: &str = "Try using the following endpoint format:
+    async fn my_endpoint(
```

Comment:
Should we fill in the endpoint name they used rather than calling it `my_endpoint`?